libs/features/spectrum2-styles/build.mjs

```diff
@@ -81,10 +105,15 @@ const updateMiloCSS = (css) => {
         const { prop } = decl;
         if (prop.startsWith('--s2')) {
           const spectrumProp = `--spectrum${prop.slice(4)}`;
-          if (spectrumProperties[selector][spectrumProp] !== undefined) {
-            decl.value = spectrumProperties[selector][spectrumProp];
+          const spectrumSelector = selector === ':root' ? '.spectrum' : selector === '.light' ? '.spectrum--light' : selector === '.dark' ? '.spectrum--dark' : null;
```

Comment:
Nested ternaries are hard to read, error-prone & discouraged by most linting configurations. Hence I'd suggest to switch to an if/else:
```suggestion
let spectrumSelector;
if (selector === ':root') {
  spectrumSelector = '.spectrum';
} else if (selector === '.light') {
  spectrumSelector = '.spectrum--light';
} else if (selector === '.dark') {
  spectrumSelector = '.spectrum--dark';
} else {
  spectrumSelector = null;
}
```

Reply:
I realized this ternary actually turned out to be redundant so its been removed after looking into this a bit more.